retry/retry.go

```diff
@@ -115,6 +115,8 @@ type Option func(r *options)
 // Max attempts of 0 indicates no limit.
 //
 // If max attempts option is not used, then default value of 0 is used.
+// If initial backoff is larger than max backoff and the max backoff is nonzero, the initial backoff will be
```

Comment:
this is the wrong place for this godoc?

Reply:
True, people should read the initialBackoff docs if they use it, so need for a dup.